tests/models/nn/transformers/test_sasrec.py

```diff
@@ -968,6 +968,8 @@ def initial_config(self) -> tp.Dict[str, tp.Any]:
             "backbone_type": TransformerTorchBackbone,
             "get_val_mask_func": leave_one_out_mask,
             "get_trainer_func": None,
+            "get_val_mask_func_kwargs": None,
```

Comment:
Please add tests for custom trainer func with kwargs and custom validation func with kwargs